internal/host/validator.go

```diff
@@ -717,16 +720,16 @@ func (v *validator) validateNetworkLatencyForRole(host *models.Host, clusterRole
 	failedHostNames := []string{}
 	for _, r := range connectivityReport.RemoteHosts {
 		for _, l3 := range r.L3Connectivity {
+			hostname, role, err := GetHostnameAndRoleByIP(l3.RemoteIPAddress, hosts)
+			if err != nil {
+				v.log.Warnf("unable to unmarshall host connectivity for %s:%s", host.ID, err)
+				return ValidationFailure, nil, err
```

Comment:
How do you suggest to handle this use case? @gamli75 confirmed that `Pending` state is only when the host needs user intervention. I don't think it makes sense to continue the execution in case of unmarshalling failure and end up returning success because the slice is empty.

Reply:
Note that QE just opened a BZ on this: https://bugzilla.redhat.com/show_bug.cgi?id=1983412
Should we then ignore the error and continue execution or return as `Pending`? /cc @gamli75

Reply:
Ok, `ValidationError` should do.

Reply:
My remark was for the error message. The connectivity unmarshall is on line 715.  Here it is a different error.

Reply:
@ori-amizur you can see here: https://github.com/openshift/assisted-service/blob/master/internal/host/validator.go#L464 
if inventory == nil - it's pending
if the unmarshal failed - it's failure

Reply:
Changed to log the error message verbatim. Also I change to return a `ValidationError` if the unmarshal of the connectivity report fails. WDYT @gamli75 @ori-amizur ?